es-core/src/components/VideoComponent.cpp

```diff
@@ -221,6 +212,15 @@ void VideoComponent::applyTheme(const std::shared_ptr<ThemeData>& theme, const s
 	if (elem->has("showSnapshotDelay"))
 		mConfig.showSnapshotDelay = elem->get<bool>("showSnapshotDelay");
 
+	setRotation(0.0);
```

Comment:
This method is only executed when loading a theme.  It is intended to ensure that the default values are set properly when switching between themes.  I will test it more to verify if it is needed or not.